python/rainbow/simulators/prox_soft_bodies/api.py

```diff
@@ -1,12 +1,14 @@
 from typing import List, Dict
 import rainbow.geometry.grid3 as GRID
 import rainbow.geometry.kdop_bvh as BVH
+from rainbow.geometry.spatial_hashing import HashGird
```

Comment:
Style issue, all other imports of own modules uses capital letters, that would be HASH_GRID

Reply:
No problem